accounts/migrations/0002_drop_profile.py

```
@@ -0,0 +1,14 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("accounts", "0001_squashed_0005_customuser_cutover"),
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)